Cargo.toml

```diff
@@ -29,13 +29,14 @@ backend-kvm = ["x86_64", "kvm-bindings", "kvm-ioctls"]
 backend-sgx = ["x86_64", "sgx", "ureq"]
 gdb = ["gdbstub"]
 dbg = []
+disable-attestation = []
 
 [dependencies]
 x86_64 = { version = "^0.14.8", default-features = false, optional = true }
-sgx = { version = "0.3.0", features = ["openssl"], optional = true }
+sgx = { git = "https://github.com/enarx/sgx", rev = "f6de78316cdd74abf825dc3f6794f6f43f744e85", features = ["openssl"], optional = true}
 const-default = { version = "1.0", features = [ "derive" ] }
 primordial = { version = "0.4", features = ["alloc"] }
-sallyport = { version = "0.3.0", git = "https://github.com/enarx/sallyport", rev = "580480b19b1542bc3da2fdafef08f895f5bb3389" }
+sallyport = { version = "0.3", git = "https://github.com/jarkkojs/sallyport", rev = "71e2342" }
```

Comment:
Could you please mark your PR as DRAFT as long as your private repo is in there?

Reply:
There's two private repos, actually. I also have a pending PR here: https://github.com/enarx/mmledger/pull/41

Reply:
converted

Reply:
So this is blocked on https://github.com/enarx/sallyport/pull/114 , which is itself being held back until after the next release, yes?